src/lib/button-toggle/button-toggle.ts

```diff
@@ -264,7 +264,7 @@ export class MatButtonToggleGroup extends _MatButtonToggleGroupMixinBase impleme
     }
 
     if (this.multiple && Array.isArray(this._rawValue)) {
-      return !!this._rawValue.find(value => toggle.value != null && value === toggle.value);
+      return null != this._rawValue.find(value => toggle.value != null && value === toggle.value);
```

Comment:
If you only care about the *presence* of the value, you can use `some` instead of `find`:
```ts
return this._rawValue.some(v => toggle.value != null && v === toggle.value);
```

Reply:
Thanks. Updated.